test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -646,6 +646,14 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			Expect(successes).To(Equal(cfg.StabilityIterations))
 		})
 
+		It("should have stable pod to wireserver connectivity", func() {
+			name := fmt.Sprintf("alpine-%s", cfg.Name)
```

Comment:
@tamilmani1989 you can wrap this in an if condition if this will only work for certain cluster configurations

e.g., if ip-masq-agent prevents container access to 168.63.129.16 on TCP 80:

`if !eng.ExpandedDefinition.Properties.IsIPMasqAgentEnabled()`

Reply:
Once @JunSun17 PR is merged, it should work for all cases. For now, we can have this condition but it should would work in all cases

Reply:
Which PR?

Reply:
Is there an AKS Engine change that this depends upon?

Reply:
the one he shared in email. Its vsts link

Reply:
ipmasqagent code is in AKS. we have dependency on that. Both PR has to be in to allow connection from pod to wireserver. otherwise ipmasqagent will rewrite those ipable rules if its enabled.